weblate/trans/vcs.py

```diff
@@ -733,7 +733,7 @@ def push(self):
         on fork and creates pull request against original repository.
         """
         self.fork()
-        fork_branch = '{0}-weblate'.format(self.branch)
+        fork_branch = '{0}-weblate'.format(self.name)
```

Comment:
This will not really improve situation as name in this case is VCS class name (GitHub).